source/install/desktop-msi-installer-and-group-policy-install.rst

```diff
@@ -120,11 +120,15 @@ Install the Mattermost Desktop App using the MSI installer
    .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00016.png
       :alt: You'll be prompted to allow the Mattermost Desktop App to make changes to your system. You must select Yes to continue with the installation process.
 
-3. Select ``Finish`` when the installation is complete.
+3. Select **Finish** when the installation is complete.
 
    .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00017.png
       :alt: When the installation is complete, select Finish.
 
+.. tip::
+
+   To prevent Mattermost from adding a desktop shortcut for the desktop app, open the MSI file with an editor, delete the desktop shortcut code, and save that change as an MST file. Install the MSI using the MST file.
```

Comment:
I'd never heard of this process. Where did it come from?

Reply:
@devinbinnie - It was from a [peer-to-peer discussion](https://community.mattermost.com/core/pl/bdchci7qd3dxmktonm74n8qika) on the Mattermost Community Server. Please let me know if this approach is something Mattermost would officially support. If it's not, happy to remove.

Reply:
Gotcha, so this isn't an officially supported use case by us, but I guess it does happen to work. 0/5 on whether we keep it or not.